tfx/extensions/google_cloud_big_query/pusher/executor.py

```diff
@@ -41,6 +41,9 @@
 _BQ_DATASET_ID_KEY = 'bq_dataset_id'
 _MODEL_NAME_KEY = 'model_name'
 
+# Project where query will be executed
+_COMPUTE_PROJECT_ID_KEY = 'compute_project_id'
```

Comment:
Could you update the docstring (line 74 see below) to include a description of what this configuration is, what is the default behaviour and possibly an example of how this configuration can be used if the user has two projects per your usecase. 
```
...
  def Do(self, input_dict: Dict[Text, List[types.Artifact]],
         output_dict: Dict[Text, List[types.Artifact]],
         exec_properties: Dict[Text, Any]):
    """Overrides the tfx_pusher_executor.
    Args:
    ...
      exec_properties: Mostly a passthrough input dict for
        tfx.components.Pusher.executor.  custom_config.bigquery_serving_args is
        consumed by this class.  For the full set of parameters supported by
        Big Query ML, refer to https://cloud.google.com/bigquery-ml/
```